### PR TITLE
feat(UI-867): user profile info in sidebar

### PR DESCRIPTION
**Pull request by J1za**
## Description

Adjustments:
1. Instead of an avatar, use this: https://www.npmjs.com/package/react-avatar to display initials (only initials, not avatar picture)
2. On click in the design we have: settings, logout, please add also the email and name like in the screenshot below:
image

## Linear Ticket
https://linear.app/autokitteh/issue/UI-867/user-profile-bubble
## What type of PR is this? (check all applicable)

-   [x] 💡 (feat) - A new feature (non-breaking change which adds functionality)
-   [ ] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
-   [ ] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
-   [ ] 🏎 (perf) - Optimization
-   [ ] 📄 (docs) - Documentation - Documentation only changes
-   [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
-   [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
-   [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

### CODE REVIEW

File: src/components/organisms/sidebar/sidebar.tsx [hunk 1]
```diff
@@ -173,47 +174,47 @@ export const Sidebar = () => {
...
-											</motion.span>
-										) : null}
-									</AnimatePresence>
-								</Button>
-							</>
+								</PopoverTrigger>
+								<PopoverContent className="z-50 min-w-56 rounded-2xl border border-gray-950 bg-white px-3.5 py-2.5 font-averta shadow-2xl">
+									<div className="flex items-center gap-2 border-b border-b-gray-950 pb-2">
+										<Avatar color="black" name={`${user?.name}`} round={true} size="32" />
+										<span className="font-medium text-black">{user?.name}</span>
```
Comment (RonenMars) [inline src/components/organisms/sidebar/sidebar.tsx:199]:
```suggestion
										<span className="font-medium text-black">{user?.email}</span>
```

File: src/components/organisms/sidebar/sidebar.tsx [hunk 2]
```diff
@@ -173,47 +174,47 @@ export const Sidebar = () => {
...
+											title={t("settings")}
+										>
+											{t("settings")}
+										</Button>
+										<Button
+											className="w-full rounded-md px-2.5 text-lg hover:bg-gray-250"
+											onClick={() => logoutFunction()}
+										>
+											{t("logout")}
+										</Button>
```
Comment (RonenMars) [inline src/components/organisms/sidebar/sidebar.tsx:214]: Please keep the icons for the buttons (settings & logout) like in the screenshot in the ticket

File: src/components/organisms/sidebar/sidebar.tsx (hunk 1 above)
Comment (J1za) [reply src/components/organisms/sidebar/sidebar.tsx:199]: changed

File: src/components/organisms/sidebar/sidebar.tsx (hunk 2 above)
Comment (J1za) [reply src/components/organisms/sidebar/sidebar.tsx:214]: added

**Review by RonenMars — APPROVED** [review]

File: src/components/organisms/sidebar/sidebar.tsx [hunk 3]
```diff
@@ -173,47 +175,49 @@ export const Sidebar = () => {
 								) : null}
 							</AnimatePresence>
 						</Button>
-						{isAuthEnabled ? (
-							<>
-								<Button className="hover:bg-green-200" href="/settings" title={t("settings")}>
-									<SettingsIcon className="size-7" fill="black" />
 
+						{!isAuthEnabled ? (
```
Comment (RonenMars) [inline src/components/organisms/sidebar/sidebar.tsx:179]:
```suggestion
						{isAuthEnabled ? (
```

Comment (J1za) [reply src/components/organisms/sidebar/sidebar.tsx:179]: changed

**Pull request merged**